content/Timer_and_Divider_Registers.md

```diff
@@ -40,6 +40,7 @@ Bits 1-0 - Input Clock Select
 ```
 
 ::: tip NOTE
+Select the clock frequency first, then enable the Timer.
```

Comment:
Do you mean that it should be done by a separate write? If so, why should that be done?

Reply:
Explained